src/partials/footer.html

```diff
@@ -1,47 +1,71 @@
-<section class="footer"> 
-    <div class="container footer__container">      
-        <div class="footer__left"> 
-            <h3 class="footer__title">homemade ice cream made</h3>
-            <h3 class="footer__title-second">with passion </h3>
-        </div>     
-        <div class="footer__right">
-            <ul class="footer__social-list">
-                <li class="footer_social-list-item">
-                    <a class="social-list__link" href="">
-                        <svg class="social-list__icon">
-                        <use href="./images/icons-footer-all.svg#icon-Vector-1"></use>
-                        </svg>
-                </a>
-                </li>
-                <li class="footer_social-list-item">
-                    <a class="social-list__link" href="">
-                        <svg class="social-list__icon">
-                            <use href="./images/icons-footer-all.svg#icon-telegram-1"></use>     
-                        </svg>
-                    </a>
-                </li>
-                <li class="footer_social-list-item">
-                    <a class="social-list__link" href="">
-                        <svg class="social-list__icon">
-                            <use href="./images/icons-footer-all.svg#icon-Vector"></use>     
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Nagłówek strony</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&family=Titan+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./footer.css">
+    <link rel="stylesheet" href="./container.css">
+    <link rel="stylesheet" href="./images/">
+
+
```

Comment:
od tej linijki w góre trzeba to pousuwać, nie może być tego w projekcie, bo wszystko się popsuje